app/src/main/java/tech/ula/model/state/SessionStartupFsm.kt

```diff
@@ -339,5 +364,7 @@ data class AssetDownloadComplete(val downloadAssetId: Long) : SessionStartupEven
 object SyncDownloadState : SessionStartupEvent()
 object CopyDownloadsToLocalStorage : SessionStartupEvent()
 data class VerifyFilesystemAssets(val filesystem: Filesystem) : SessionStartupEvent()
+object VerifyAvailableStorage : SessionStartupEvent()
+object VerifyAvailableStorageComplete : SessionStartupEvent()
```

Comment:
I think `VerifyAvailableStorageComplete` and the associated logic could be removed from the FSM. Rather than wait for that event here and emit the success state for this stage, I think the VM could directly submit the `ExtractFilesystem` event when the `VM#lowAvailbleStorageAcknowledged` callback is made.